shared/src/simulation/events/patient-category-transfer-to-hospital-finished.ts

```diff
@@ -0,0 +1,38 @@
+import { IsBoolean } from 'class-validator';
+import { getCreate } from '../../models/utils';
+import { IsLiteralUnion, IsValue } from '../../utils/validators';
+import {
+    AlivePatientTriageCategory,
+    alivePatientTriageCategoryAllowedValues,
+} from '../utils/alive-patient-triage-category';
+import type { SimulationEvent } from './simulation-event';
+
+export class PatientCategoryTransferToHospitalFinishedEvent
+    implements SimulationEvent
+{
+    @IsValue('patientCategoryTransferToHospitalFinishedEvent')
+    readonly type = 'patientCategoryTransferToHospitalFinishedEvent';
+
+    @IsLiteralUnion(alivePatientTriageCategoryAllowedValues)
+    readonly patientCategory: AlivePatientTriageCategory;
+
+    /**
+     * This is true, if this refers to its own one single region.
+     * This is false, if it refers to all regions managed by one behavior.
+     */
+    @IsBoolean()
+    readonly isRelatedToOwnRegion: boolean;
```

Comment:
In my opinion, either these cases should be split into separate events or the name must be more descriptive, like `ownRegionOnly`.

Reply:
Its now called `isRelatedOnlyToOwnRegion`